--- /etc/grub.d/10_linux	2020-05-11 02:44:27.000000000 -0600
+++ /etc/grub.d/10_linux_modified	2020-05-17 13:50:24.890662484 -0600
@@ -267,6 +267,8 @@
     linux_entry "${OS}" "${version}" simple \
     "${GRUB_CMDLINE_LINUX} ${GRUB_CMDLINE_LINUX_DEFAULT}"
 
+    linux_entry "Windows 10" "${version}" simple "boot_windows=1 win_boot_entry=0000"
+
     submenu_indentation="$grub_tab"
     
     if [ -z "$boot_device_id" ]; then
